testutil/testchain.go

```diff
@@ -8,6 +8,7 @@ import (
 	blocks "github.com/ipfs/go-block-format"
 	cid "github.com/ipfs/go-cid"
 	"github.com/ipld/go-ipld-prime"
+	_ "github.com/ipld/go-ipld-prime/codec/dagcbor"
```

Comment:
sounds fine. oy. I wonder if we should just preload them in the root package at some point

Reply:
my initial take was to put them both there - but when you use graphsync as a whole (not just as a unit test) you're going to get dagcbor no matter what because it's in the messaging layer. `raw` is going to come if you need it, but that's up to the user IMO, maybe they want to add dagjson, dagjose, git, and more, but that's up to them so I'm fine with localising this I think.

Reply:
cool.